package.json

```diff
@@ -13,68 +13,55 @@
     "integration:local": "npm run serve",
     "integration:metamask": "npm run integration:wait && npm run integration:transfers",
     "integration:wait": "wait-on -l -i 1000 tcp:8545 http-get://localhost:5000",
-    "integration:transfers": "node ./integration/index.mjs",
+    "integration:transfers": "node --experimental-modules ./integration/index.mjs",
```

Comment:
no need experimental modules, it's on by default on node 12

Reply:
Require --experimental-modules flag in order for the GitHub ci to run, else will have loader error. However I found other script error of parsing argument into test script to set slowMo in PR #2